easybuild/tools/options.py

```diff
@@ -383,6 +383,8 @@ def override_options(self):
                                      None, 'store_true', False),
             'zip-logs': ("Zip logs that are copied to install directory, using specified command",
                          None, 'store_or_None', 'gzip'),
+            'set-as-default': ("Set the generated module as default", None, 'store_true', False),
```

Comment:
maybe rename to `--set-default-module`? leaves less room for misinterpretation...